BigCommerce/Integration/02_Particularities.md

```diff
@@ -0,0 +1,190 @@
+   
+For example, an offer is created by using the defining attributes *Size* and *Color*. By using these attributes, two variants are created &ndash; T-shirt-s-red (S, red) and t-shirt-m-green (M, green). The driver creates the following options and values:
+- Size (S, M),
+- Color (Green, Red)  
+
+The *BigCommerce driver* of *Actindo* creates only the variants "t-shirt-s-red" and "t-shirt-m-green", but the *BigCommerce* storefront presents all potential options and value pairings. This includes products such as t-shirt-s-green (S, green), which do not exist. These product variants are not only visible to the customers, but they can also order these non-existing products.  
+
+**To do**:   
+You must manage this issue by the storefront itself.
+In *BigCommerce*, you can access via the **/v3/catalog/products/{productId}/variants** API path all variants the driver has created.
+
+
+
+## Import of offers 
+
+The following particularities are relevant when you initially import *BigCommerce* products to the offers in the *Omni-Channel* module.
+
+
+### Shared variant options
+
+*BigCommerce* has shared variant options that allow you to easily create variants once and then apply them to multiple products with a single click. These shared variants cannot be modified in any way using the API: 
+- The driver can import offers with shared variant options, but you are not able to edit them after the import.   
+- The driver cannot manage any changes after the import in the *BigCommerce* UI. Changing shared variant options after an import will cause *Omni-Channel* offer updates to fail.
+   
+
+**To do:**   
+Before the first import of offers, check your variant options in *BigCommerce*. It is strongly recommended not to use shared variant options. To check it, see your *BigCommerce* store under *Products > Product options > Shared options tab*. Remove any shared option that might be available there.
+
+![Shared option tab in BigCommerce](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/BigCommerceSharedOptions.png "[Shared option tab in BigCommerce]")
+
+Instead, you can use the variants you can add to a product or an offer in the *PIM* or the *Omni-Channel* module.
+
+
+
+## Import of orders 
+
+The following particularities are relevant when you import your orders from *BigCommerce* to the *Omni-Channel* module.
+
+The import of offers is supported by the *BigCommerce* driver.
+- You can filter the orders that are to be imported. For detailed information, see [Configure BigCommerce connection](./01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).  
+- The *BigCommerce* connection enables parallel imports to handle larger loads.
```

Comment:
Does it? I believe currently we import synchronously.

Reply:
Satz rausgenommen